src/providers/seventv/SeventvEventAPI.cpp

```diff
@@ -35,6 +39,19 @@ void SeventvEventAPI::subscribeUser(const QString &userID,
     }
 }
 
+void SeventvEventAPI::subscribeTwitchChannel(const QString &id)
+{
+    if (this->subscribedTwitchChannels_.insert(id).second)
+    {
+        this->subscribe(
+            {ChannelCondition{id}, SubscriptionType::CreateCosmetic});
+        this->subscribe(
+            {ChannelCondition{id}, SubscriptionType::CreateEntitlement});
+        this->subscribe(
+            {ChannelCondition{id}, SubscriptionType::DeleteEntitlement});
```

Comment:
cosmetic nitpick: add a `,` after the last parameter in the unsubscribe call so it takes more lines and looks nicer